test/unittests/tools/files/test_zipping.py

```diff
@@ -109,6 +109,22 @@ def test_untargz():
     assert exists(join(dest_dir, "src", "bar.txt"))
 
 
+def test_untargz_output():
```

Comment:
Better add a check in some existing tests (I guess it is likely that some existing test will break because of this change, didn't any test fail when running locally?)

Reply:
Changed the check to an existing test. I didn't run all the test suite locally, but the test I run were just fine 😄